iphone/cli/commands/_build.js

```diff
@@ -412,6 +412,10 @@ iOSBuilder.prototype.config = function config(logger, config, cli) {
 							default: true,
 							desc: __('focus the iOS Simulator')
 						},
+						'export-ipa': {
```

Comment:
Yes and no. Yes to the first part. No to the second part. You need to the xarchive because that's what you're exporting the .ipa from.